modules/nf-core/sambamba/sort/main.nf

```diff
@@ -0,0 +1,38 @@
+process SAMBAMBA_SORT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::sambamba=1.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity//sambamba:1.0--h98b6b92_0':
+    'quay.io/biocontainers/sambamba:1.0--h98b6b92_0' }"
+
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
```

Comment:
Can this module output CRAM files?

Reply:
Nope, it is not mentioned in the documentation of sambamba sort.